tgui/packages/tgui/interfaces/OperatingComputer.js

```diff
@@ -0,0 +1,534 @@
+                  key={organ_bundle["organ_name"]}
+                  bundle={organ_bundle}
+                />
+              );
+            })
+          }
+        </Table>
+      </Stack.Item>
+    ); }
+};
+
+const DisplayOrgan = (props) => {
+  const { bundle } = props;
+  const organ_special = bundle["special"];
+  const organ_state = bundle["organ_state"];
+  const organ_name = bundle["organ_name"];
+
+  let font_color = "green";
+  let is_bold = false;
+
+  if (organ_state === "Missing") {
+    font_color = "red";
+    is_bold = true;
+  }
+  if (organ_state === "Dead") {
+    font_color = "red";
+    is_bold = true;
+  }
+  if (organ_state === "Critical") {
+    font_color = "orange";
+    is_bold = true;
+  }
+  if (organ_state === "Significant") {
+    font_color = "orange";
+  }
+  if (organ_state === "Moderate") {
+    font_color = "yellow";
+  }
+
+  if (organ_state !== "Okay" || organ_special) {
+    return (
+      <Table.Row>
+        <Table.Cell header textAlign="right" width={10}>
+          {capitalize(spaceUnderscores(organ_name))}:
+        </Table.Cell>
+        <Table.Cell width={10} color={font_color} bold={is_bold}>
+          {organ_state !== "Okay" ? organ_state : "" }
+          {organ_special ? <Box color="white">{organ_special}</Box>: "" }
+        </Table.Cell>
+      </Table.Row>
+    );
+  }
+};
+
+const DisplayLimbs = (props, context) => {
+  const { limb_status } = props;
+  const { data } = useBackend(context);
+  if (data.occupied) {
+    return (
+      <Stack.Item width={20}>
+        <Table>
+          <Table.Row>
+            <Table.Cell header textAlign="right">Limb</Table.Cell>
+            <Table.Cell header>Status</Table.Cell>
+          </Table.Row>
+          {
+            limb_status.map((limb_bundle) => {
+              return (
+                <DisplayLimb
+                  key={limb_bundle["limb"]}
+                  bundle={limb_bundle}
+                />
+              );
+            })
+          }
+        </Table>
+      </Stack.Item>
+    );
+  }
+};
+
+const DisplayLimb = (props) => {
+  const { bundle } = props;
+  const limb_name = bundle["limb"];
+  const limb_status = bundle["status"];
+  let limb_color = "white";
+  let is_bold = false;
+
+  if (limb_status === "Missing") {
+    is_bold = true;
+    limb_color = "red";
+  }
+
+  if (limb_status !== "Okay") {
+    return (
+      <Table.Row>
+        <Table.Cell header textAlign="right" width={10}>
+          {capitalize(spaceUnderscores(limb_name))}:
+        </Table.Cell>
+        <Table.Cell width={10} color={limb_color} bold={is_bold}>{limb_status}</Table.Cell>
+      </Table.Row>
+    );
+  }
+};
+
+const DisplayTemperature = (props, context) => {
+  const { data } = useBackend(context);
+  const { body_temp, optimal_temp } = props;
+  let font_color = "grey";
+  let body_temp_c = "--";
+  let body_temp_f = "--";
+  if (data.occupied) {
+    if (body_temp >= (optimal_temp + 60)) { font_color="red"; }
```

Comment:
we stole this from a dm proc, so we should put it back and make both the original source give us data and color we can use directly.